src/main/java/frc/robot/subsystems/snapback/shooter/ShooterIO.java

```diff
@@ -17,12 +17,16 @@ public static class ShooterIOInputs {
     public double leftAppliedVolts;
     public double leftCurrentAmps;
     public double leftTemperatureCelsius;
```

Comment:
Need to add input for left goal velocity
```suggestion
    public double leftVelocityGoalRadiansPerSecond;
```